Gemfile

```diff
@@ -15,3 +15,7 @@ end
 group :test, :development do
   gem 'pry-byebug'
 end
+
+group :test do
+  gem 'rspec', '>= 3.2.0'
```

Comment:
Do we still need mini-test or that can be replaced with rspec? If we want to keep them, we have to move it from development to test section